src/components/Forms/useForm.js

```diff
@@ -32,9 +34,31 @@ const useForm = validate => {
             ...values,
             [name]: value
         });
-    };
+    }
     const handleSubmit = e => {
-        e.preventDefault()
+        e.preventDefault();
+        axios.post(url, {
+            firstName: values.firstName,
+            lastName: values.lastName,
+            company: values.comapny,
+            phone: values.phone,
+            email: values.email,
+            confirmemail: values.confirmemail,
+            password: values.password,
+            confirmpassword: values.confirmpassword,
+            address: values.address,
+            state: values.state,
+            postcode: values.postcode,
+            showPassword: false,
+            licence:values.licence,
+            role:values.role,
+            categories:values.categories,
+            description: values.description,
+
```

Comment:
looking up [the swagger](http://taskstech2.pythonanywhere.com/swagger/#/users/post_users_customer)

- create a customer
{
  "email": "string",
  "password": "string",
  "first_name": "string",
  "last_name": "string",
  "phone": "string",
  "address": "string"
}
- create tradeperson
{
  "email": "string",
  "password": "string",
  "first_name": "string",
  "last_name": "string",
  "description": "string",
  "phone": "string"
}

maybe change the payload of post req,  ONLY including those fields at the time of creating a new tradeperson and customer